UI/js-src/lsmb/users/ChangePassword.js

```diff
@@ -30,7 +30,7 @@ define([
             _lstrings: {
                 title: "Change Password",
                 "old password": "Old Password",
-                "new password": "New password",
+                "new password": "New Password",
```

Comment:
This changes a translated string, which means see need to update the translation template file. Please include a run of "make pot".

Reply:
A fix is needed so that `make pot` parses multiple `text(..)` strings
placed on a single line, which occurs in this case. I'll do that and submit
the results in a separate PR.

For now, the highlighted string is not included in translations, so we're no
worse off.